packages/avocode-email-tagsinput/test/e2e/avocode-email-tagsinput.spec.js

```diff
@@ -0,0 +1,127 @@
+import puppeteer from 'puppeteer'
+import { expect } from 'chai'
+import {
+  URL,
+  tagsInputFieldSelector,
+  getAnchorAndFocus,
+  addEmailTag,
+  getTagNodes, getInputText,
+} from './test-utils'
+
+describe('Basic Email Tags Input', () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch(browserOptions)
+  })
+
+  beforeEach(async () => {
+    page = await browser.newPage()
+    await page.goto(URL)
+  })
+
+  afterEach(async () => {
+    await page.close()
+  })
+
+  afterAll(async () => {
+    await browser.close()
+  })
+
+  const navigateToUnique = async () => {
+    const elements = await page.$x('//a[@href="#/avocode-email-tagsinput/unique"]')
+    await elements[0].click()
```

Comment:
This is not nice modifying DOM anyway. You'd want to do redirect. 

Also this is what I mean in my previous review when I said it'd be better to create special testing routes in `playground` package just for testing purposes. This is just unnecessary complexity. Don't do it in this PR though, just a food for thought how to do it next time.